Bottom: f1461e27a310bcba5bc27c10f61f02e952bdffd2
Top:    b2a9a2f17408095d906fc886ecb278d1c7370d17
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-16 15:29:34 +0800

XDP: introduce XDP_HOLD


---

diff --git a/include/uapi/linux/bpf.h b/include/uapi/linux/bpf.h
index 7d9b283..4e177aa 100644
--- a/include/uapi/linux/bpf.h
+++ b/include/uapi/linux/bpf.h
@@ -530,6 +530,7 @@ enum xdp_action {
 	XDP_DROP,
 	XDP_PASS,
 	XDP_TX,
+	XDP_HOLD,
 };
 
 /* user accessible metadata for XDP packet hook
